.travis.yml

```diff
@@ -31,36 +31,45 @@ matrix:
+        - JOB="build"
+        - CODECOV="true"
     - name: Test Kyuubi w/ -Pspark-3.0 and Spark 3.1 binary
       env:
-        - PROFILE="-Pspark-3.0 -Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.1.1 -Dspark.archive.name=spark-3.1.1-bin-hadoop2.7.tgz"
-        - EXCLUDE_TAGS="org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DataLakeTest"
-    - name: Test Kyuubi w/ -Pspark-3.1
+        - PROFILE="-Pspark-3.0 -Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.1.1 -Dspark.archive.name=spark-3.1.1-bin-hadoop2.7.tgz -Dmaven.plugin.scalatest.exclude.tags=org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DeltaTest,org.apache.kyuubi.tags.IcebergTest"
+        - JOB="build"
+    - name: Test Kyuubi w/ -Pspark-3.1 -Pkyuubi-extension-spark_3.1
       env:
-        - PROFILE="-Pspark-3.1"
-        - EXCLUDE_TAGS="org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DataLakeTest"
-    - name: Test Kyuubi w/ -Pspark-3.1 -Pspark-hadoop-3.2
-      env:
-        - PROFILE="-Pspark-3.1 -Dspark-hadoop-3.2"
-        - EXCLUDE_TAGS="org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DataLakeTest"
+        - PROFILE="-Pspark-3.1 -Pkyuubi-extension-spark_3.1"
+        - JOB="build"
     - name: Test Kyuubi w/ -Pspark-3.2-snapshot w/ Spark 3.2 nightly build
       env:
-        - PROFILE="-Pspark-3.2-snapshot -pl :kyuubi-spark-sql-engine,:kyuubi-common,:kyuubi-ha,:kyuubi-zookeeper"
-        - EXCLUDE_TAGS="org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DataLakeTest"
+        - PROFILE="-Pspark-3.2-snapshot"
+        - JOB="nightly-build"
     - name: Test Kyuubi w/ -Pspark-3.0 TPCDS Tests
       env:
-        - TPCDS_TESTS=true
+        - PROFILE="-Pspark-3.0"
+        - JOB="tpcds-tests"
 
 install:
   - mvn --version
 
 script:
-  - if [[ "$TPCDS_TESTS" == "true" ]]; then mvn clean install -DskipTests -pl :kyuubi-spark-sql-engine,:kyuubi-common,:kyuubi-ha,:kyuubi-zookeeper && mvn -Dtest=none -DwildcardSuites=org.apache.kyuubi.operation.tpcds.TPCDSOutputSchemaSuite,org.apache.kyuubi.operation.tpcds.TPCDSDDLSuite -Dmaven.plugin.scalatest.exclude.tags="" test; fi
-  - if [[ "$TPCDS_TESTS" != "true" ]]; then mvn clean install $PROFILE -Dmaven.plugin.scalatest.exclude.tags=$EXCLUDE_TAGS -Dmaven.javadoc.skip=true -V ;fi
+  - |
+    if [[ "$JOB" == "build" ]]; then
+      mvn clean install $PROFILE -Dmaven.javadoc.skip=true -V
+    fi
+  - |
+    if [[ "$JOB" == "nightly-build" ]]; then
+      mvn clean install $PROFILE -pl :kyuubi-spark-sql-engine,:kyuubi-common,:kyuubi-ha,:kyuubi-zookeeper -Dmaven.javadoc.skip=true -V
+    fi
+  - |
+    if [[ "$JOB" == "tpcds-tests" ]]; then
+      mvn clean install $PROFILE -DskipTests -pl :kyuubi-spark-sql-engine,:kyuubi-common,:kyuubi-ha,:kyuubi-zookeeper
+      mvn test $PROFILE -Dtest=none -DwildcardSuites=org.apache.kyuubi.operation.tpcds.TPCDSOutputSchemaSuite,org.apache.kyuubi.operation.tpcds.TPCDSDDLSuite
```

Comment:
org.apache.kyuubi.operation.tpcds.TPCDSOutputSchemaSuite,org.apache.kyuubi.operation.tpcds.TPCDSDDLSuite

can we rename them to xxxTPCDSSuite then `-DwildcardSuites=*TPCDSSuite` for short?